cookbook-react/src/recipes/radios/zecatDifapro/index.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { staticProps } from './mock';
+
+import ZecatDifapro from '.';
+
+jest.mock('i18next', () => ({
+  t: (key: string) => key
+}));
+
+describe('ZecatDifapro test', () => {
```

Comment:
Add a screenshot of the component's test coverage. You can get a report using `npm run coverage`. It should be 100%